docs/cluster_management.md

```diff
@@ -2,45 +2,76 @@
+To start using Kubernetes you need an access to machine(s) of one of supported configurations. You can see the status of support of various environments in [Getting Started Guide](../docs/getting-started-guides/README.md).
 
-## Updgrading to a different API version
+## Creating a Cluster
 
-There is a sequence of steps to upgrade to a new API version.
+Assuming that you have machines for your clusters ready, you need to start running Kubernetes on them. Detailed instructions can be found in Getting started guide:
+* GKE [GKE page](https://cloud.google.com/container-engine),
+* Vagrant [Vagrant setup](../docs/getting-started-guides/vagrant.md),
+* GCE [GCE setup](../docs/getting-started-guides/gce.md),
+* CoreOS on Azure [CoreOS on Azure setup](../docs/getting-started-guides/coreos/azure/README.md),
+* CoreOS on AWS [CoreOS on AWS setup](../docs/getting-started-guides/coreos.md),
+* CoreOS on VMWare [CoreOS on VMWare setup](../docs/getting-started-guides/coreos.md).
 
-1. Turn on the new api version
-2. Upgrade the cluster's storage to use the new version.
-3. Upgrade all config files. Identify users of the old api version endpoints.
-4. Update existing objects in the storage to new version by running cluster/update-storage-objects.sh
-3. Turn off the old version.
+## Configuration options
 
-### Turn on or off an API version for your cluster
+Different environments may have different configurations options, so here we provide only most general and by far not exhaustive configuration options of Kubernetes cluster. Configuration of Kubernetes components (e.g. API server, Kubelet) is passed as commandline flags to the binaries. If environment of your choice is using Salt to setup the cluster you can check how we pass them to the components in `cluster/saltbase/salt` subdirectories.
 
-Specific API versions can be turned on or off by passing --runtime-config=api/<version> flag while bringing up the server. For example: to turn off v1 API, pass --runtime-config=api/v1=false.
-runtime-config also supports 2 special keys: api/all and api/legacy to control all and legacy APIs respectively. For example, for turning off all api versions except v1, pass --runtime-config=api/all=false,api/v1=true.
+Two interesting examples:
+* `max-pods` in Kubelet setting maximal number of Pods that are allowed to run on a single Node,
+* `pod-eviction-timeout` in kube-controller-manager setting how long we wait after Node becomes unresponsive before we start evicting Pods from it.
+More can be seen by running particulars app help.
 
-### Switching your cluster's storage API version
+If you want to change the value of particular flag, e.g. `max-pod`, you need to find a file used by Salt to set Kubelet flags. This particular file is `cluster/saltbase/salt/kubelet/default`. In this file you can modify the line with DAEMON_ARGS by adding the value for the flag you want, e.g. change the DAEMON_ARGS line to
```

Comment:
We should be careful not to spec too much about salt.  Only some of the platforms use salt.  Also, we should not replicate too much of the cluster turnup docs into here.  Did you take a look at Eric's new 'from scratch' guide?  Can we remove anything here in favor of putting it there?